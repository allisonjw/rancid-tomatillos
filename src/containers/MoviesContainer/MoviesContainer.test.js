// import React from 'react';
// import { render } from '@testing-library/react';
// import { shallow } from 'enzyme';
// import MoviesContainer from './MoviesContainer';
//
//
// describe('MovieRatings', () => {
//   let wrapper;
//
//   beforeEach(() => {
//
//     wrapper = shallow(
//         <MoviesContainer
//         />
//     )
//
//   })
//
//   it('should render with correct data', () => {
//     expect(wrapper).toMatchSnapshot()
//   })
//
// })
