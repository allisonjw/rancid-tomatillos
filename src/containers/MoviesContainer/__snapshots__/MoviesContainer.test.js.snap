// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesContainer should take a snapshot of the data coming in 1`] = `
<Fragment>
  <h1>
    Rancid Tomatillos
  </h1>
  <section
    className="moviesContainer"
  >
    <MoviesCard
      average_rating={4}
      backdrop_path="https://image.tmdb.org/t/p/original//zTxHf9iIOCqRbxvl8W5QYKrsMLq.jpg"
      id={1}
      key="1"
      overview="In Jumanji: The Next Level, the gang is back but the game has changed. As they return to rescue one of their own, the players will have to brave parts unknown from arid deserts to snowy mountains, to escape the world's most dangerous game."
      poster_path="https://image.tmdb.org/t/p/original//l4iknLOenijaB85Zyb5SxH1gGz8.jpg"
      release_date="2019-12-04"
      title="Jumanji: The Next Level"
    />
  </section>
</Fragment>
`;
